### PR TITLE
Ensure QUnit tests have a message

### PR DESCRIPTION
**Pull request by mikepack**
I don't know that we need the complexity of #233, but we should at least produce a message in the case that it's not present.

### CODE REVIEW

File: app/assets/javascripts/teaspoon/qunit.coffee [hunk 1]
```diff
@@ -33,6 +33,7 @@ class Teaspoon.Spec
     return [] unless @spec.failed
     for item in @spec.assertions
       continue if item.result
+      item.message ||= "Expected #{JSON.stringify(item.actual)} to equal #{JSON.stringify(item.expected)}"
```
Comment (jejacks0n) [inline app/assets/javascripts/teaspoon/qunit.coffee]: I haven't configured coffeescript/javascript stuff on hound yet, so feel free to ignore.

**Comment by mikepack** [comment]
Worth noting: QUnit produces decent error messages in other cases, just not with `#equal`.

File: app/assets/javascripts/teaspoon/qunit.coffee (hunk 1 above)
Comment (jejacks0n) [inline app/assets/javascripts/teaspoon/qunit.coffee]: is there an easy way to test this? these tests live in test/javascript

**Comment by mikepack** [comment]
@jejacks0n tests added

**Comment by chancancode** [comment]
:heart_eyes: @mikepack Thank you for doing this!

**Comment by mikepack** [comment]
@chancancode not a problem, thanks for all your work on this. We'll be releasing 0.9 soon, and this change will go out with that release.

**Pull request merged**